randovania/games/super_metroid/logic_database/Maridia.txt

```diff
@@ -112,7 +112,40 @@ Extra - asset_id: 126
   > Door to Maridia Map Room
       Trivial
   > Door to Crab Tunnel
-      Gravity Suit and Morph Ball and Space Jump
+      Any of the following:
+          All of the following:
+              # With Gravity
+              Gravity Suit
+              Any of the following:
+                  # Get Up
+                  Space Jump or Gravity Jump (Intermediate)
+                  # WJ
+                  Hi-Jump Boots and Wall Jumps (Beginner)
+                  # IBJ
+                  Vertical Infinite Bomb Jump (Intermediate) and Can Lay Bombs
+              Any of the following:
+                  # Through the Morphball Tunnel
+                  All of the following:
+                      # With Morph
+                      Morph Ball
+                      Any of the following:
+                          Mid-Air Morph (Intermediate)
+                          Ice Beam and Ice Beam Abuse (Advanced) and Can Use Spring Ball
+                  # Without Morph: Clip with Ice Beam Abuse
```

Comment:
some video or further description would be helpful

Reply:
The youtube video in the comment at the top of this AND tree shows the trick in full. Each of these additional requirements make it slightly different, but they're all different versions of that same tech.

Reply:
i dont see a yt vide at the top of this AND.

Reply:
Added the video and timestamped the ice clip